rancher-config/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '2'
+
+services:
+    image: nhsuk/pa11y-dashboard:${DOCKER_IMAGE_TAG}
+    database:
```

Comment:
I don't think this `database` section is required as it isn't going to be running a local db when running in rancher.